tests/execution.rs

```diff
@@ -2814,6 +2868,10 @@ fn test_err_instruction_count_syscall_capped() {
 
 #[test]
 fn test_err_non_terminate_capped() {
```

Comment:
I think the syscalls are irrelevant to these two tests and can be removed here.

Reply:
Do you mean `test_err_instruction_count_syscall_capped`? Isn't the sys call necessary there for the test?

Reply:
I mean `test_err_non_terminate_capped`